### DISCUSSION TITLE
《绝区零》青衣角色演示｜青童与白叟

### DISCUSSION THREAD

**Discussion by bytwp** [share121/inter-knot#1096] [其他]
「枪打出头鸟，破鼓重锤敲」
「欲上青天揽明月，花落，知多少？」
……
「再练练吧。」

青衣CV
中：皛四白
日：阿保麻莉亚

—— 欢迎来到新艾利都！——
《绝区零》1.1版本「卧底蓝调」将于2024年8月14日正式上线！
bilibili安卓端下载地址：https://www.biligame.com/detail/?id=108434
bilibiliPC端下载地址：https://www.biligame.com/detail/?id=111210

详细地址：【《绝区零》青衣角色展示 | 青童与白叟-哔哩哔哩】 https://b23.tv/AMrwXgg

**Comment by ZhuYuan-148** [comment]
前辈？何时来的

**Reply by bytwp** [reply]
丸辣，视频发不了还上传了

**Comment by Kamisato100** [comment]
封面错误ww
